src/cnguts.cpp

```diff
@@ -918,15 +918,19 @@ RC FC cgRddInit(	// Perform initialization common to main simulation run and eac
 
 	RC rc = RCOK;
 
+// Note: weather and TDV files set up in tp_FazInit()
+
 // Initialize for location
 	CSE_EF( Top.tp_LocInit() )	// calls slinit() w wthr file location data (via Top).
 								// inits Locsolar (comments below)
 								// CSE_EF: return RCBAD on any non-RCOK return.
 
-// Set up weather file, TDV file, and DESCONDs
-//   Do each phase, allowing file or location changes.
-//   Messages errors.
-	CSE_EF( Top.tp_WthrInit() )	// also uses Top .tp_wfName, .skyModel [,.isDT]
```

Comment:
Was this partly redundant?

Reply:
Yes.  The function actually did not do what the comments said.